visualization/include/pcl/visualization/point_picking_event.h

```diff
@@ -42,6 +42,11 @@
 #include <vector>
 
 #include <vtkCommand.h>
+#include <vtkActor.h>
```

Comment:
Please reorder the includes to be:
* PCL
* VTK
* C++ stdlib

separated by a blank line each